app/renderer/js/main.js

```diff
@@ -147,6 +149,20 @@ class ServerManagerView {
 		}
 	}
 
+	initEnterpriseSettings() {
+		const filepath = ConfigUtil.getConfigItem('enterpriseFilepath', null);
```

Comment:
The whole chunk here is wrong; we want something like
```javascript
const possibleFilePaths = [
  '/etc/zulip-desktop/enterprise.json', // mac and linux
  'C://Program Files/Zulip-Desktop-Config' // windows
].map(filepath => path.resolve(filepath);
```
and then check each file path by looping through the each of them. Also import the node.js path module here to make sure it parse the file path so they are cross-platform safe.